wally-registry-frontend/src/pages/Package.tsx

```diff
@@ -173,150 +179,188 @@ export default function Package() {
     <>
       <ContentSection>
-        {isLoaded ? (
-          isError ? (
-            <NotFoundMessage errorMessage="HTTP 404: Resource Not Found" />
-          ) : (
-            <FlexColumns>
-              <WideColumn>
-                <Heading>{packageName}</Heading>
-
-                <Paragraph>
-                  {packageMetadata?.package.description
-                    ? packageMetadata?.package.description
-                    : `${capitalize(
-                        packageMetadata?.package.name
-                      )} has no provided description.`}
-                </Paragraph>
-              </WideColumn>
-              <NarrowColumn>
-                <MetaHeader>Metadata</MetaHeader>
-
-                {packageMetadata?.package && (
-                  <MetaItem title="Install" width="full">
-                    <CopyCode
-                      packageName={packageMetadata?.package.name}
-                      version={packageMetadata?.package.version}
-                    />
-                  </MetaItem>
-                )}
-
-                <MetaItem title="Version" width="half">
-                  <select
-                    name="version"
-                    id="version-select"
-                    value={packageVersion || "?.?.?"}
-                    onChange={(a) => {
-                      hist.push(
-                        `/package/${packageScope}/${packageName}?version=${a.target.value}`
-                      )
-                    }}
-                  >
-                    {packageHistory?.map((item: WallyPackageMetadata) => {
-                      return (
-                        <option
-                          key={item.package.version}
-                          value={item.package.version}
-                        >
-                          {item.package.version}
-                        </option>
-                      )
-                    })}
-                  </select>
-                </MetaItem>
-
-                {packageMetadata?.package.license && (
-                  <MetaItem title="License" width="half">
-                    <a
-                      href={`https://choosealicense.com/licenses/${packageMetadata?.package.license.toLocaleLowerCase()}`}
+        <FlexColumns>
+          <WideColumn>
+            <Heading>{packageName}</Heading>
+
+            <Paragraph>
+              {packageMetadata?.package.description
+                ? packageMetadata.package.description
+                : `${capitalize(packageName)} has no provided description.`}
+            </Paragraph>
+          </WideColumn>
+
+          <NarrowColumn>
+            <MetaHeader>Metadata</MetaHeader>
+
+            {packageMetadata?.package && (
+              <MetaItem title="Install" width="full">
+                <CopyCode
+                  packageName={packageMetadata.package.name}
+                  version={packageMetadata.package.version}
+                />
+              </MetaItem>
+            )}
```

Comment:
Simplify since packageMetadata cannot be null here:
```suggestion
            <MetaItem title="Install" width="full">
              <CopyCode
                packageName={packageMetadata.package.name}
                version={packageMetadata.package.version}
              />
            </MetaItem>
```